operator/integration-tests/model-in-image/run-test.sh

```diff
@@ -75,7 +75,7 @@ function usage() {
                 Note that this depends on the db being deployed
                 and initialized via rcu. So either pre-deploy
                 the db and run rcu or pass '-db' and '-rcu' too.
-    -cm       : Run the tests in common mounts mode.
+    -aux      : Run the tests in auxiliary images mode.
```

Comment:
`-aux` should be `-ai` per the command line argument processing below

Reply:
Looks like it... Thanks.